### PR TITLE
Fix all instances of broken links for "Cultural Settings"

### PR DESCRIPTION
**Pull request by e0i**
via https://github.com/MicrosoftDocs/windows-powershell-docs/pull/1797

FYI @dariomws @get-itips @tseward

This is a follow-up on https://github.com/MicrosoftDocs/windows-powershell-docs/pull/1797 and #1798, via Sean's [comment](https://github.com/MicrosoftDocs/windows-powershell-docs/pull/1797#issuecomment-605343833) pointing out to correct link.

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Style checks needed for this PR. Thanks.

**Comment by e0i** [comment]
Closes #1795 #1728 


Thank you.

**Comment by get-itips** [comment]
@e0i Onur I think if, we are going to use that link, you should remove _en-us_ from all instances of the link.

**Comment by traya1** [comment]
Hi Team,

I noticed the "Admin Review" label was applied to this PR. Is there anything I can assist with?

**Comment by e0i** [comment]
> Hi Team,
> 
> I noticed the "Admin Review" label was applied to this PR. Is there anything I can assist with?

@traya1 Hello, I actually intended to use the label to indicate that the PR is being reviewed by our team. Would you please create an appropriate label if that is not OK? Thank you.

**Comment by traya1** [comment]
Personally I see no problem if you use the label though I'll see if it is required. For now, if you decide to use the label, maybe just assign someone to the PR as a placeholder so I know my help is not needed?

**Comment by e0i** [comment]
We'll need to have some repository configurations sorted out in order to tag independent reviewers. I am therefore opting out of using the "Admin Reviewer" label because it won't be needed once we do the config. Thank you for the suggestion.

File: docset/windows/international/Get-WinUserLanguageList.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Get-WinUserLanguageList [<CommonParameters>]
 ## DESCRIPTION
 The **Get-WinUserLanguageList** cmdlet returns the current user language settings.
 These settings include input method, spelling setting, text prediction setting, and handwriting input mode.
-For more information, see [CultureInfo Class](http://go.microsoft.com/fwlink/?LinkID=242306) and [Configurable Language and Cultural Settings](http://go.microsoft.com/fwlink/?LinkID=242307).
+For more information, see [CultureInfo Class](http://go.microsoft.com/fwlink/?LinkID=242306) and [Manage the input and display language settings in Windows 10](https://support.microsoft.com/help/4496404/windows-10-manage-the-input-and-display-language#input_language)
```
Comment (JohanFreelancer9) [inline docset/windows/international/Get-WinUserLanguageList.md]:
```suggestion
For more information, see [CultureInfo Class](http://go.microsoft.com/fwlink/?LinkID=242306) and [Manage the input and display language settings in Windows 10](https://support.microsoft.com/help/4496404/windows-10-manage-the-input-and-display-language#input_language).
```

File: docset/winserver2012-ps/internationalcmdlets/Get-WinUserLanguageList.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Get-WinUserLanguageList [<CommonParameters>]
 ## DESCRIPTION
 The **Get-WinUserLanguageList** cmdlet returns the current user language settings.
 These settings include input method, spelling setting, text prediction setting, and handwriting input mode.
-For more information, see CultureInfo Classhttp://go.microsoft.com/fwlink/?LinkID=242306 and Configurable Language and Cultural Settingshttp://go.microsoft.com/fwlink/?LinkID=242307.
+For more information, see [CultureInfo Class](http://go.microsoft.com/fwlink/?LinkID=242306) and [Manage the input and display language settings in Windows 10](https://support.microsoft.com/help/4496404/windows-10-manage-the-input-and-display-language#input_language)
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/internationalcmdlets/Get-WinUserLanguageList.md]:
```suggestion
For more information, see [CultureInfo Class](http://go.microsoft.com/fwlink/?LinkID=242306) and [Manage the input and display language settings in Windows 10](https://support.microsoft.com/help/4496404/windows-10-manage-the-input-and-display-language#input_language).
```

File: docset/winserver2012-ps/internationalcmdlets/New-WinUserLanguageList.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ New-WinUserLanguageList [-Language] <String> [<CommonParameters>]
 ## DESCRIPTION
 The **New-WinUserLanguageList** cmdlet creates a new user language list object.
 The object settings include input method, spelling setting, text prediction setting, and handwriting input mode.
-For more information, see CultureInfo Classhttp://go.microsoft.com/fwlink/?LinkID=242306 and Configurable Language and Cultural Settingshttp://go.microsoft.com/fwlink/?LinkID=242307.
+For more information, see [CultureInfo Class](http://go.microsoft.com/fwlink/?LinkID=242306) and [Manage the input and display language settings in Windows 10](https://support.microsoft.com/help/4496404/windows-10-manage-the-input-and-display-language#input_language)
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/internationalcmdlets/New-WinUserLanguageList.md]:
```suggestion
For more information, see [CultureInfo Class](http://go.microsoft.com/fwlink/?LinkID=242306) and [Manage the input and display language settings in Windows 10](https://support.microsoft.com/help/4496404/windows-10-manage-the-input-and-display-language#input_language).
```

File: docset/winserver2012-ps/internationalcmdlets/Set-Culture.md [hunk 4]
```diff
@@ -23,7 +23,7 @@ Set-Culture [-CultureInfo] <CultureInfo> [<CommonParameters>]
 ## DESCRIPTION
 The **Set-Culture** cmdlet sets a specific culture (also known as a locale for unmanaged code development) for the current user account.
 The information includes the names for the culture, the writing system, the calendar, and formatting for dates and sort strings.
-For more information, see CultureInfo Classhttp://go.microsoft.com/fwlink/?LinkID=242306 and Configurable Language and Cultural Settingshttp://go.microsoft.com/fwlink/?LinkID=242307.
+For more information, see [CultureInfo Class](http://go.microsoft.com/fwlink/?LinkID=242306) and [Manage the input and display language settings in Windows 10](https://support.microsoft.com/help/4496404/windows-10-manage-the-input-and-display-language#input_language)
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/internationalcmdlets/Set-Culture.md]:
```suggestion
For more information, see [CultureInfo Class](http://go.microsoft.com/fwlink/?LinkID=242306) and [Manage the input and display language settings in Windows 10](https://support.microsoft.com/help/4496404/windows-10-manage-the-input-and-display-language#input_language).
```

File: docset/winserver2012-ps/internationalcmdlets/Set-WinUserLanguageList.md [hunk 5]
```diff
@@ -25,7 +25,7 @@ Set-WinUserLanguageList
 ## DESCRIPTION
 The **Set-WinUserLanguageList** cmdlet sets the current user language settings.
 These settings include input method, spelling setting, text prediction setting, and handwriting input mode.
-For more information, see CultureInfo Classhttp://go.microsoft.com/fwlink/?LinkID=242306 and Configurable Language and Cultural Settingshttp://go.microsoft.com/fwlink/?LinkID=242307.
+For more information, see [CultureInfo Class](http://go.microsoft.com/fwlink/?LinkID=242306) and [Manage the input and display language settings in Windows 10](https://support.microsoft.com/help/4496404/windows-10-manage-the-input-and-display-language#input_language)
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/internationalcmdlets/Set-WinUserLanguageList.md]:
```suggestion
For more information, see [CultureInfo Class](http://go.microsoft.com/fwlink/?LinkID=242306) and [Manage the input and display language settings in Windows 10](https://support.microsoft.com/help/4496404/windows-10-manage-the-input-and-display-language#input_language).
```

File: docset/winserver2012r2-ps/international/Get-WinUserLanguageList.md [hunk 6]
```diff
@@ -31,7 +31,7 @@ Get-WinUserLanguageList [<CommonParameters>]
 ## DESCRIPTION
 The **Get-WinUserLanguageList** cmdlet returns the current user language settings.
 These settings include input method, spelling setting, text prediction setting, and handwriting input mode.
-For more information, see CultureInfo Classhttp://go.microsoft.com/fwlink/?LinkID=242306 and Configurable Language and Cultural Settingshttp://go.microsoft.com/fwlink/?LinkID=242307.
+For more information, see [CultureInfo Class](http://go.microsoft.com/fwlink/?LinkID=242306) and [Manage the input and display language settings in Windows 10](https://support.microsoft.com/help/4496404/windows-10-manage-the-input-and-display-language#input_language)
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/international/Get-WinUserLanguageList.md]:
```suggestion
For more information, see [CultureInfo Class](http://go.microsoft.com/fwlink/?LinkID=242306) and [Manage the input and display language settings in Windows 10](https://support.microsoft.com/help/4496404/windows-10-manage-the-input-and-display-language#input_language).
```

File: docset/winserver2012r2-ps/international/New-WinUserLanguageList.md [hunk 7]
```diff
@@ -31,7 +31,7 @@ New-WinUserLanguageList [-Language] <String> [<CommonParameters>]
 ## DESCRIPTION
 The **New-WinUserLanguageList** cmdlet creates a new user language list object.
 The object settings include input method, spelling setting, text prediction setting, and handwriting input mode.
-For more information, see CultureInfo Classhttp://go.microsoft.com/fwlink/?LinkID=242306 and Configurable Language and Cultural Settingshttp://go.microsoft.com/fwlink/?LinkID=242307.
+For more information, see [CultureInfo Class](http://go.microsoft.com/fwlink/?LinkID=242306) and [Manage the input and display language settings in Windows 10](https://support.microsoft.com/help/4496404/windows-10-manage-the-input-and-display-language#input_language)
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/international/New-WinUserLanguageList.md]:
```suggestion
For more information, see [CultureInfo Class](http://go.microsoft.com/fwlink/?LinkID=242306) and [Manage the input and display language settings in Windows 10](https://support.microsoft.com/help/4496404/windows-10-manage-the-input-and-display-language#input_language).
```

File: docset/winserver2012r2-ps/international/Set-Culture.md [hunk 8]
```diff
@@ -31,7 +31,7 @@ Set-Culture [-CultureInfo] <CultureInfo> [<CommonParameters>]
 ## DESCRIPTION
 The **Set-Culture** cmdlet sets a specific culture (also known as a locale for unmanaged code development) for the current user account.
 The information includes the names for the culture, the writing system, the calendar, and formatting for dates and sort strings.
-For more information, see CultureInfo Classhttp://go.microsoft.com/fwlink/?LinkID=242306 and Configurable Language and Cultural Settingshttp://go.microsoft.com/fwlink/?LinkID=242307.
+For more information, see [CultureInfo Class](http://go.microsoft.com/fwlink/?LinkID=242306) and [Manage the input and display language settings in Windows 10](https://support.microsoft.com/help/4496404/windows-10-manage-the-input-and-display-language#input_language)
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/international/Set-Culture.md]:
```suggestion
For more information, see [CultureInfo Class](http://go.microsoft.com/fwlink/?LinkID=242306) and [Manage the input and display language settings in Windows 10](https://support.microsoft.com/help/4496404/windows-10-manage-the-input-and-display-language#input_language).
```

File: docset/winserver2012r2-ps/international/Set-WinUserLanguageList.md [hunk 9]
```diff
@@ -33,7 +33,7 @@ Set-WinUserLanguageList
 ## DESCRIPTION
 The **Set-WinUserLanguageList** cmdlet sets the current user language settings.
 These settings include input method, spelling setting, text prediction setting, and handwriting input mode.
-For more information, see CultureInfo Classhttp://go.microsoft.com/fwlink/?LinkID=242306 and Configurable Language and Cultural Settingshttp://go.microsoft.com/fwlink/?LinkID=242307.
+For more information, see [CultureInfo Class](http://go.microsoft.com/fwlink/?LinkID=242306) and [Manage the input and display language settings in Windows 10](https://support.microsoft.com/help/4496404/windows-10-manage-the-input-and-display-language#input_language)
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/international/Set-WinUserLanguageList.md]:
```suggestion
For more information, see [CultureInfo Class](http://go.microsoft.com/fwlink/?LinkID=242306) and [Manage the input and display language settings in Windows 10](https://support.microsoft.com/help/4496404/windows-10-manage-the-input-and-display-language#input_language).
```

**Comment by vazome** [comment]
The link ["Configurable Language and Cultural Settings"](https://msdn.microsoft.com/Areas/Epx/Content/500.aspx?aspxerrorpath=/en-us/globalization/mt662310) is still not available.

**Comment by e0i** [comment]
@vazome It seems the changes have not been published yet, as the branch has not been merged to the `live` branch during scheduled merge. It will be available within the next scheduled merge. Thank you for the notice.

**Comment by e0i** [comment]
Also fixes #1220

**Pull request merged**